vignettes/unit_test_guidance.Rmd

```diff
@@ -211,6 +211,110 @@ knitr::include_graphics("./unit_test_toc.png")
 ```
 
 
+## Addin `pharmaverse4devs::format_test_that_file()`
+
+To ease the burden on developers for writing and adding tests we have developed an Addin for formatting test_that test files according to admiral programming standards. The Addin will add and update comments as well as number or re-numbers the tests. To access the Addin, be sure to install the {pharmaverse4devs} from Github. 
+
+To install the latest development version of the package directly from
+GitHub use the following code:
+```r
+    if (!requireNamespace("remotes", quietly = TRUE)) {
+      install.packages("remotes")
+    }
+
+    remotes::install_github("pharmaverse/pharmaverse4devs")
+```
+
+Then use the Addin button and select the "Format
+test_that test file" as seen in the image.  Be sure to have the test-file open and selected when calling the Addin.
+
+```{r echo=FALSE, out.width='120%'}
+knitr::include_graphics("./unit_test_format_tests.png")
+```
+
+The Addin will perform the following:
+
+- Updates or adds the number of the tests in the comments and in the
+`test_that()` call
+- Updates the comments based on the description provided in the `test_that()`
+call
+- Updates the function name in the `test_that()` call. The function name is
+extracted from the last `# <function name> ----` comment before the
+`test_that()` call. If a test file tests more than one function, such comments
+should be added before the first test of each function. If a test files tests a
+single function only, the comments can be omitted. In this case the addin
+determines the function name from the file name by stripping of the "test-"
+prefix and the ".R" suffix.
+
+When writing new unit tests, just provide a description in the `test_that()`
+call and if necessary the function name in a `# <function name> ----` comment:
+```
+# arg_name ----
+test_that("arg_name works", {
+  expect_equal(arg_name(sym("a")), "a")
+  expect_equal(arg_name(call("enquo", sym("a"))), "a")
+  expect_error(arg_name("a"), "Could not extract argument name from")
+})
+
+# convert_dtm_to_dtc ----
+test_that("works if dtm is in correct format", {
+  expect_equal(
+    convert_dtm_to_dtc(as.POSIXct("2022-04-05 15:34:07 UTC")),
+    "2022-04-05T15:34:07"
+  )
+})
+
+test_that("Error is thrown if dtm is not in correct format", {
+  expect_error(
+    convert_dtm_to_dtc("2022-04-05T15:26:14"),
+    "lubridate::is.instant(dtm) is not TRUE",
+    fixed = TRUE
+  )
+})
+```
+
+Call the addin and get:
+```
+# arg_name ----
+## Test 1: arg_name works ----
+test_that("arg_name Test 1: arg_name works", {
+  expect_equal(arg_name(sym("a")), "a")
+  expect_equal(arg_name(call("enquo", sym("a"))), "a")
+  expect_error(arg_name("a"), "Could not extract argument name from")
+})
```

Comment:
can you swap out for another example - this will complete https://github.com/pharmaverse/admiraldev/issues/436 issue as arg_name is no longer available